torch/csrc/jit/runtime/profiling_record.h

```diff
@@ -184,15 +184,11 @@ struct ProfilingRecord {
   std::shared_ptr<Graph> profiled_graph_;
   mutable std::mutex mutex_;
   size_t profiling_count_;
-  // the key is a frame id
-  // the value is a mapping from a Value in a graph
-  // to a profiled TensorType
-  std::map<int64_t, std::map<Value*, TensorTypePtr>> profiled_types_per_frame_;
 
   // A thin wrapper around `partitionSetByDimension` to ensure
   // `new_sizes` and `sym_shapes` are of the same rank
 
-  c10::SymbolicShape mergeSymbolicShapes(
+  static c10::SymbolicShape mergeSymbolicShapes(
```

Comment:
I was hoping we can nuke this helper? Are there any uses?

Reply:
whoops, missed that - I think you're right